apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -129,7 +129,7 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         template,
         readOnly,
         urlSearchParams,
-        [ "data-testid" ]: testId
+        ["data-testid"]: testId
```

Comment:
Add a config for your VSCode to not to autoformat `tsx|jsx|js|ts` files. Revert all other white space removals.

Reply:
Fixed